modules/rest-http/src/test/java/org/apache/ignite/internal/processors/rest/protocols/http/jetty/GridRestSuite.java

```diff
@@ -24,7 +24,8 @@
  */
 @RunWith(Suite.class)
 @Suite.SuiteClasses({
-    RestSetupSimpleTest.class
+    RestSetupSimpleTest.class,
```

Comment:
Do we run this suite anywhere? I guess we don't.

Let's find where the rest of REST tests are (ignite-clients?), move it there.

Reply:
This is where REST is started. Other REST tests don't actually start http-rest module.